diff --git a/dialog/store/DialogStore.js b/dialog/store/DialogStore.js
index 24a4165..31c7eb4 100644
--- a/dialog/store/DialogStore.js
+++ b/dialog/store/DialogStore.js
@@ -74,7 +74,6 @@ function createInitialState(initialState = {}) {
     ...(0, _popups.createInitialPopupStoreState)(),
     modal: true,
     disablePointerDismissal: false,
-    popupElement: null,
     viewportElement: null,
     descriptionElementId: undefined,
     titleElementId: undefined,
diff --git a/esm/dialog/store/DialogStore.js b/esm/dialog/store/DialogStore.js
index c85b62c..6c8d6e8 100644
--- a/esm/dialog/store/DialogStore.js
+++ b/esm/dialog/store/DialogStore.js
@@ -66,7 +66,6 @@ function createInitialState(initialState = {}) {
     ...createInitialPopupStoreState(),
     modal: true,
     disablePointerDismissal: false,
-    popupElement: null,
     viewportElement: null,
     descriptionElementId: undefined,
     titleElementId: undefined,
